src/main/java/com/ea/restaurant/service/OrderStatusHistoryService.java

```diff
@@ -0,0 +1,10 @@
+package com.ea.restaurant.service;
+
+import com.ea.restaurant.entities.OrderStatusHistory;
+import java.util.List;
+
+public interface OrderStatusHistoryService {
+  List<OrderStatusHistory> findLastOrderStatusHistoriesByOrderIds(List<Long> orderIds);
+
+  void InsertNewOrUpdatedBatchOrderStatusHistories(List<OrderStatusHistory> orderStatusHistories);
```

Comment:
Rename to `create`

Reply:
No, this method insert order status histories after update, it is not a create method.